s3-proxy/s3-proxy/src/main/java/com/woowacourse/s3proxy/exception/UploadFailException.java

```diff
@@ -0,0 +1,14 @@
+package com.woowacourse.s3proxy.exception;
+
+public class UploadFailException extends PickGitStorageException {
+
+    private static final String MESSAGE = "업로드 실패";
```

Comment:
```suggestion
    private static final String ERROR_CODE = "U0001";
```

메시지 대신 에러 코드로 작성해주세요 !